src/main/java/racing/domain/Judge.java

```diff
@@ -0,0 +1,32 @@
+package racing.domain;
+
+import racing.domain.RacingCar;
+import racing.domain.RacingCars;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Judge {
+    public static List<String> selectWinners(RacingCars racingCars) {
+        int highScore = getMaxPosition(racingCars);
```

Comment:
```suggestion
          int highScore = racingCars.getMaxPosition();
```

상태를 가진 객체에게 물어봐도 좋지 않을까요?  😄

Reply:
가장 멀리 이동한 위치를 구하는 로직을 RacingCars 내부로 이동하였습니다.